Source/macOS/OIDRedirectHTTPHandler.m

```diff
@@ -91,6 +92,11 @@ - (NSURL *)startHTTPListener:(NSError **)returnError {
   return nil;
 }
 
+- (NSURL *)startHTTPListener:(NSError **)returnError {
+  //a port of 0 dictactes a random port
```

Comment:
nit, I'd change:
`//a port of 0 dictactes a random port`
to
`// A port of 0 requests a random available port`